### PR TITLE
Add accessiblity helpers for Happychat

### PR DESCRIPTION
**Pull request by mkaz**
Addresses some issues around accessibility for Happychat

### CODE REVIEW

File: client/components/happychat/composer.jsx [hunk 1]
```diff
@@ -44,7 +44,11 @@ export const Composer = React.createClass( {
 				onMouseLeave={ this.scrollbleedUnlock }
 				>
 				<div className="happychat__message">
+					<label htmlFor="hc_message_textarea" style={{ display: "none"}}>
```
Comment (jasmussen) [inline client/components/happychat/composer.jsx]: Can't you just do `<label><textarea ...></label>`? Then you shouldn't need to add any IDs or invisible-making styles.

Comment (mkaz) [inline client/components/happychat/composer.jsx]: My understanding is that it needs some text to read what the textarea is for.
It looks like an alternate could be to use `aria-label` within the textarea
See: https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html

Comment (jasmussen) [inline client/components/happychat/composer.jsx]: Ah, I see what you're doing now. Yep, based on the description for the aria label, I think that's what we should do, but now that I understand what you were trying to do I could get behind the other version as well. My preference is for the visual appearance to be as close to the aria appearance as possible, though.

**Comment by jasmussen** [comment]
Looks good with the latest aria change!

**Pull request merged**